onnxruntime/core/providers/qnn/builder/onnx_ctx_model_helper.cc

```diff
@@ -33,30 +34,26 @@ bool GraphHasEpContextNode(const onnxruntime::GraphViewer& graph_viewer) {
 }
 
 bool IsFusedGraphHasCtxNode(const std::vector<IExecutionProvider::FusedNodeAndGraph>& fused_nodes_and_graphs) {
+  bool has_qnn_ep_context_node = false;
   for (const auto& fused_node_graph : fused_nodes_and_graphs) {
     const onnxruntime::GraphViewer& graph_viewer(fused_node_graph.filtered_graph);
-    bool has_qnn_ep_context_node = GraphHasEpContextNode(graph_viewer);
+    has_qnn_ep_context_node = GraphHasEpContextNode(graph_viewer);
     if (has_qnn_ep_context_node) {
       return true;
     }
   }
-  return false;
+  return has_qnn_ep_context_node;
```

Comment:
Q: The functionality of this function hasn't changed (returns true as soon as a graph has a context node). I just want to double check that this wasn't meant to do something else.

Reply:
Changed this method in one of the commit but realized it was useless, and tried to revert it. But seems didn't revert it to the original code. Will clean it up.